build/tradable.mjs

```diff
@@ -2,6 +2,17 @@ const builtUntradable = ['Warframe', 'Throwing', 'Shotgun', 'Rifle', 'Pistol', '
 const tradableConditions = (item) => !(builtUntradable.includes(item.type) && item.name.match(/Prime/gi));
 
 const tradableTypes = [
```

Comment:
I updated this in master to split out a list of tradable mods, could you do the same for an array of `tradableArcanes` and spread them onto the `tradableTypes` array?